libs/sdk-ui-filters/src/AttributeFilter/utils/AttributeFilterUtils.ts

```diff
@@ -0,0 +1,29 @@
+// (C) 2021 GoodData Corporation
+
+import { IAttributeElement } from "@gooddata/sdk-backend-spi";
+
+export const getAllTitleIntl = (intl: any, isInverted: boolean, empty: boolean, equal: boolean) => {
```

Comment:
The `intl` can be typed properly with some type from "react-intl" please use it.

Reply:
`IntlShape` type used